docs: redoing for what was pr 4672 (#4676)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/release-notes/2.2.1/index.md

### ClusterResourceSet deployments create an unbounded number of service-account-tokens (COPS-7267)

> An issue with the ClusterResourceSet controller in 2.2.0 caused an unbounded number of service account tokens to be created for each ClusterResourceSet. The problem has been corrected. A remediation is also available to identify and remove the excess secrets; see this [knowledge base](https://support.d2iq.com/hc/en-us/articles/6019137621908-Customer-Advisory-D2iQ-2022-0002-Unbounded-Number-of-Service-Account-Token-Secrets-Created) article for more information.

Reviewer: ```suggestion
An issue with the ClusterResourceSet controller in 2.2.0 causes an unbounded number of service account tokens to be created for each ClusterResourceSet. This problem has been corrected. A remediation is also available to identify and remove the excess secrets; see this [knowledge base](https://support.d2iq.com/hc/en-us/articles/6019137621908-Customer-Advisory-D2iQ-2022-0002-Unbounded-Number-of-Service-Account-Token-Secrets-Created) article for more information.
```

- kommanderAppManagementImageRepository
- kommanderChartsVersion

> If any of the these fields are present, then there is a possibility the upgrade can fail. If you encounter this situation, file a support ticket for advice on how to remediate the issue before attempting to continue the upgrade.

Reviewer: ```suggestion
If any of the these fields are present, the upgrade can fail. If you encounter this situation, contact D2iQ Support for advice on how to remediate the issue before attempting to continue the upgrade.
```

### Minio Disk insufficient space when upgrading

When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, we recommend that you disable the `fluent-bit` Platform Application before upgrading.

> ### Calico not updated during DKP upgrade

Author: ```suggestion
### Calico not updated during DKP upgrade on Flatcar
```

> When upgrading a DKP cluster, after what seems to be a successful upgrade, the Calico service might not update as expected and, therefor, is still using the old image. The wrong CNI ClusterResourceSet is being generated and not accounting for Flatcar. This issue only impacts Calico, no other add-ons.

Reviewer: See notes above and change this to be in line with other item.
